core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -878,6 +878,35 @@ private String parsePath(String bucketPath) throws S3Exception {
     return normalizedBucket;
   }
 
+  private String parsePath(String bucketPath, String prefix, String delimiter) throws S3Exception {
+    // Alluxio only support use / as delimiter
+    if (!delimiter.equals(AlluxioURI.SEPARATOR)) {
```

Comment:
In this version, we only support / as a delimiter in Alluxio, will check if we have any scenarios also need other delimiters